pkg/ccl/backupccl/restore_job.go

```diff
@@ -394,7 +393,12 @@ func restore(
 
 			if idx >= mu.ceiling {
 				for i := mu.ceiling; i <= idx; i++ {
-					importSpan := <-importSpanCh
+					importSpan, ok := <-importSpanCh
```

Comment:
Note to others who were momentarily confused by importSpanCh being read both here and in spanCountTask: this variable is defined, captured by the func, and then redefined, so it is actually two different channels.